src/core/io/ibprof_plain.c

```diff
@@ -62,13 +62,13 @@ void ibprof_io_plain_dump(FILE* file, IBPROF_OBJECT *ibprof_obj)
 
 			_ibprof_module_dump(file, temp_module_obj, ibprof_obj->hash_obj, ibprof_obj->task_obj->procid);
 
-			total_time_in_ms = ibprof_hash_module_total(ibprof_obj->hash_obj,
+			total_time = ibprof_hash_module_total(ibprof_obj->hash_obj,
 				temp_module_obj->id,
 				ibprof_obj->task_obj->procid);
 
-			plain_output(file, "%-30.30s :    %20.4f\n", "total", total_time_in_ms);
+			plain_output(file, "%-30.30s :    %20.4f\n", "total", total_time);
 			plain_output(file, DELIMITER);
-			plain_output(file, "%-30.30s :    %20.4f %\n", "wall time (%)", total_time_in_ms
+			plain_output(file, "%-30.30s :    %20.4f %\n", "wall time (%)", total_time
 				/ (ibprof_obj->task_obj->wall_time * 1.0e+6));
```

Comment:
@liranoz12 could you check if this calculation correct. As I understand total_time can be in different measurement values basing on user setting but wall_time is in sec.